modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// storage/logical-volume-manager-storage.adoc
+
+:_content-type: CONCEPT
+[id="lvms-preface-sno-ran_{context}"]
+= Deploying logical volume manager storage on {sno} clusters using {rh-rhacm}
+
+You can deploy the logical volume manager storage on a {sno} bare-metal or user provisioned infrastructure cluster and configure it to dynamically provision storage for your workloads.
+
+The logical volume manager storage creates a volume group using all the available unused disks and creates a single thin pool with a size of 90% of the volume group.
```

Comment:
There are other instances of this in the doc but `logical volume manager storage` is the product name so can we drop `the`? My thinking is, we don't say `The OpenShift` only `OpenShift` so I'd expect the same syntax to apply with `logical volume manager storage`.